packages/splitter/splitter.stories.tsx

```diff
@@ -52,33 +54,34 @@ export const Container = ({
   max,
   orientation,
   ariaLabel,
-  primaryPaneId,
   defaultValueNow,
   keyboardStep,
-  rtl
+  placement
 }) => {
   const Splitter = () => {
-    const mode = rtl ? 'rtl' : 'ltr';
+    const mode = placement === SplitterPlacement.START ? 'rtl' : 'ltr';
```

Comment:
The story behavior for `orientation="horizontal" placement="start"` appears to be broken. We might need to more closely examine what some of this means and see if non-sensical combos can be mitigated in the hook.

Reply:
Given offline meeting, we will reintroduce `rtl` prop and add `position` prop with `before | after` properties back into the Splitter props. Ill verify that behavior remains consistent for horizontal and 'before' positioning.

Reply:
I think the name `placement` doesn't work well for determining writing direction layout. For example, in `orientation="horizontal"` the UI flips from top to bottom (vise versa). Since RTL is a writing direction, UI only flips from `start` to `end` (or vise versa). Which leads me to believe the prop should be named `rtl`, similar to the other the hooks in the project. Besides this hook, would we refactor the argument name for other hooks in this project from `rtl` -> `placement`? I am curious what you both think @mtomcal @jzempel.

Reply:
Agreed, Henry – current naming fights the mental model. #namingishard.  A couple of contenders might be:
- boolean prop (which more accurately lines-up with the rest of Garden) `isTrailingPrimary`/`isLeadingPrimary` based on what we think the more prevalent use case will be.
- a different enum, such as `position="lead | trail"

I'm not trying to name on the fly; just leaving food for thought. Please use these comments as fodder to reconsider the naming here.

Reply:
I refactored `placement` to `position` with an enum type of `SeparatorPosition.LEADS | SeparatorPosition.TRAILS` and reintroduced an `rtl` prop to keep `position` relative to reading direction. https://github.com/zendeskgarden/react-containers/pull/405/commits/1fcd3fe466056a77363624600aee432203f87351